public/app/features/alerting/unified/components/receivers/ReceiversTable.tsx

```diff
@@ -301,7 +303,10 @@ export const ReceiversTable = ({ config, alertManagerName }: Props) => {
   const [showCannotDeleteReceiverModal, setShowCannotDeleteReceiverModal] = useState(false);
 
   const isGrafanaAM = alertManagerName === GRAFANA_RULES_SOURCE_NAME;
-  const showExport = isGrafanaAM && contextSrv.hasAccess(permissions.provisioning.read, isOrgAdmin());
+  const showExport =
+    isGrafanaAM &&
+    (contextSrv.hasAccess(permissions.provisioning.read, isOrgAdmin()) ||
```

Comment:
I think we need to do this for [notification policy export in Policy.tsx](https://github.com/grafana/grafana/pull/70009/files#diff-f26fd61ed962ab09579374b8b948aee0c02617f634d35d29e9af7762dfcd79c6) as well.